### DISCUSSION TITLE
The language of all my files is not recognized, like ts, js, vue...

### DISCUSSION THREAD

**Discussion by DrAugus** [github-linguist/linguist#5946] [Q&A] [answered]
Maybe [folder `.vuepress`](https://github.com/DrAugus/draugus.github.io/tree/master/docs/.vuepress) excluded…

repo is [here](https://github.com/DrAugus/draugus.github.io)

![image](https://user-images.githubusercontent.com/34206897/174724808-9f5cbd35-afd7-4c11-bd76-8cf185cc4e94.png)

**Comment by lildude** [comment, marked as answer]
This is because all your code is under the `docs` directory which is commonly used for… documentation 😁 so Linguist ignores it and everything within it:

https://github.com/github/linguist/blob/d7799da826e01acdb8f84694d33116dccaabe9c2/lib/linguist/documentation.yml#L12

You can implement an override to tell Linguist to not consider the directory as documentation.
